starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +15,39 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(list) {
```

Comment:
Good solution!!! :tada: But sometimes a for loop is more efficient. For example:
```
function findLongestWord(arr) {
  if (arr.length === 0) {
    return undefined;
  }
  var longestWord = "";
  for (i = 0; i < arr.length; i++) {
    if (arr[i].length > longestWord.length) {
      longestWord = arr[i];
    }
  }
  return longestWord;
}